<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Tabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>911</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Centerline">
     <attribute name="title">
      <string>Centerline</string>
     </attribute>
     <widget class="QPushButton" name="loadMapButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
      <property name="text">
       <string>Load map...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SaveCenterLineButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Save centerline ...</string>
      </property>
     </widget>
     <widget class="QSlider" name="thresholdSlider">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>190</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>10</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="sliderPosition">
       <number>50</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
      <property name="tickInterval">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="thresholdLabel">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>170</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Threshold</string>
      </property>
     </widget>
     <widget class="QLabel" name="thresholdValue">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>190</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0.5</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="reverseCheckBox">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>120</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Reverse</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QGraphicsView" name="centerLineView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>661</width>
        <height>521</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="genCenterlineButton">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>240</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Raceline">
     <attribute name="title">
      <string>Raceline</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Compare">
     <attribute name="title">
      <string>Compare</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>907</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHello">
    <property name="title">
     <string>Hello</string>
    </property>
   </widget>
   <addaction name="menuHello"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoadMap">
   <property name="text">
    <string>LoadMap</string>
   </property>
   <property name="toolTip">
    <string>Click here to load a .pgm file</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
